### PR TITLE
LPS-125175 Keep features, related to the upload field for guests users, only in Forms

### PR DESCRIPTION
**Pull request by carolmariaabb**
https://issues.liferay.com/browse/LPS-125175

### CODE REVIEW

**Comment by carolmariaabb** [comment]
ci:test:sf

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:test:sf

**Comment by carolmariaabb** [comment]
ci:test:relevant

File: modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/utils/settingsForm.es.js [hunk 1]
```diff
@@ -89,6 +89,14 @@ export const getFilteredSettingsContext = ({
 						] = getPredefinedValues(field);
 					}
 
+					if (fieldName == 'repeatable') {
```
Comment (natocesarrego) [inline modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/utils/settingsForm.es.js:92]: @carolmariaabb, is there any reason for not use the operator `===` instead?

If not, can you please use it?

Comment (carolmariaabb) [reply modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/utils/settingsForm.es.js]: No reason, I just didn't pay attention 😅 

I resent it here. Thanks!

**Comment by carolmariaabb** [comment]
ci:test:sf

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Comment by natocesarrego** [comment]
ci:forward

**Pull request closed without merging**